core/base/integralLines/IntegralLines.h

```diff
@@ -282,13 +290,10 @@ int ttk::IntegralLines::execute(Compare cmp) const {
 
   {
     std::stringstream msg;
-    msg << "[IntegralLines] Data-set (" << vertexNumber_
-        << " points) processed in " << t.getElapsedTime() << " s. ("
-        << threadNumber_ << " thread(s))." << std::endl;
-    dMsg(std::cout, msg.str(), timeMsg);
+    msg << "Data-set (" << vertexNumber_ << " points) processed in "
+        << t.getElapsedTime() << " s. (" << threadNumber_ << " thread(s)).";
+    this->printMsg(msg.str());
```

Comment:
same remark as before (both timing and thread reporting can be reported following ttk's convention).